DEVELOPING.md

```diff
@@ -0,0 +1,45 @@
+# Developing ros-cross-compile
+
+This document is intended to contain instructions, guidelines, and requirements for developing this tool.
+
+## Writing Code
+
+It is recommended to use `virtualenv` to manage the dependency environment.
+
+```
+# create a virtualenv
+virtualenv venv
+# use it
+source venv/bin/activate
+# install this package in the virtualenv to get dependencies
+pip install -e .
+# now you can run your work
+ros_cross_compile
+```
+
+## Tests
+
+There are currently two test entrypoints
+
+* Unit tests via `tox`
+  * run `tox -e py`
+* End-to-end shell script
+  * run `./test/run_e2e_test.sh`
+
+## Host Platforms
+
+This tool aims to support as host platforms
+* all ROS 2 Tier 1 platforms
+* on x86_64 (cross-compiling _from_ ARM platforms has not been deemed necessary)
+* for LTS distributions that have not hit EOL
```

Comment:
Let's clearly state what we support instead of having three bullets.

This tool supports all Tier-1 platform targeted by non-EOL ROS 2 distributions

What about ROS 1?

Reply:
Not going to include ROS 1. It's a closely overlapping set, but adds more surface area (e.g. Melodic supports Ubuntu 17). Noetic will be Tier 1 on Ubuntu 20.04 - we will support that.